build/test-scripts/ios-uitest-run.sh

```diff
@@ -198,6 +198,11 @@ then
 		fi
 	done
 
+	if ! [ -f "$SIMCTL_CHILD_UITEST_RUNTIME_AUTOSTART_RESULT_FILE" ]; then
+ 		echo "The file $SIMCTL_CHILD_UITEST_RUNTIME_AUTOSTART_RESULT_FILE is not available, waiting 2 seconds"
+ 		sleep 2
+ 	fi
```

Comment:
If we ever see `"The file /tmp/TestResult-*.xml is not available, waiting 2 seconds"` followed by `"The file /tmp/TestResult-*.xml is available, the test run is complete."`, that means the extra sleep is helping.